calendars/matplotlib.yaml

```
@@ -17,3 +17,21 @@ events:
         # seconds, minutes, hours, days, weeks, months, years
         days: 7
       until: 2025-01-01 00:00:00
+
+  - summary: Monthly Matplotlib New Contributor Meeting
+    description: |
+      The monthly call for new contributors to the Matplotlib project 🎉
+
+      All are welcome.
+
+      Meeting notes: https://hackmd.io/@matplotlib
+
+    begin: 2022-05-10 19:00:00
+    end: 2022-05-10 20:00:00
+    url: https://us06web.zoom.us/j/81285851006?pwd=Tks2QjRkNWh5NGw0TmU1RUwwOVluZz09
+    repeat:
+      interval:
+        # seconds, minutes, hours, days, weeks, months, years
+        days: 28
+      until: 2025-01-01 00:00:00 +00:00
```

Maybe this is the issue
```suggestion
      until: 2025-01-01 00:00:00
```

---

I was going to say this is what we have for the NumPy meetings, but indeed now there is a timezone option which is maybe conflicting with this note here

---

If we want multiple time zone support that's something we could add in our lib.